javascript/javascript1/week4/voiceassistant.js

```diff
@@ -0,0 +1,54 @@
+let toDoList = [];
+
+function getToday() {
+  let today = new Date();
+  let day = today.getDate();
+  let month = today.getMonth() + 1;
+  let year = today.getFullYear();
+  let marr = new Array(
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December"
+  );
+  console.log(day + " of " + marr[month] + " " + year);
+}
+
+function myFunction() {
+  console.log("Timer Done");
+}
+
+function getReply(command) {
+  if (command == "Hello my name is Benjamin") {
+    console.log("nice to meet you Benjamin");
+  } else if (command == "Add fishing to my todo") {
+    toDoList.push("fishing");
```

Comment:
using == operator to match string is not so appropriate. include()  is gives most accurate result. works for all cases.

Try to use different string functions like startsWith(), endsWith()  to check fist and last matching words. 
in this case if string starts with add and ends with todo, then you should add fishing to lsit.